docs/tutorial.md

```diff
@@ -260,7 +260,7 @@ Note that the version of the `@prisma/photon` and `prisma2` packages must always
 
 To keep a migration history, Prisma by default uses a folder called `migrations`. There are two ways how the `migrations` folder gets populated:
 
-- Whenever the data model in _development mode_, a new migration is generated into `migrations/dev`.
+- Whenever the data model ... in _development mode_, a new migration is generated into `migrations/dev`.
 - Whenever a data model change is to be persisted using Lift, it gets its own directory.
```

Comment:
It seems like words are missing here but I wasn't 100% what... "whenever the data model ... in development mode".
Any suggestions? I think it could be a little confusing as an incomplete sentence.

Reply:
"Whenever the data model is run in development mode, ..." perhaps?